controller.inc.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class ObjController {
+
+  function ObjController() {} //constructor
+
+  function valid($x, $y, $f) { //Validate inputs.
```

Comment:
You need to learn about [DocBlocks](http://docs.phpdoc.org/guides/docblocks.html)
It helps your colleagues to understand methods in and outs and is also link to some editors handy functionnalities